src/main/java/city/parking/exceptions/ParkingProcessNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package city.parking.exceptions;
+
+public class ParkingProcessNotFoundException extends RuntimeException {
```

Comment:
Jakie konsekwencje ma użycie RuntimeException zamiast Exception?

Reply:
Exception trzeba obsłużyć gdzieś w kodzie i wymusza to kompilator (np. bez try/catch takiego wyjątku kod się nie skompiluje, a w przypadku RuntimeException tego sprawdzenia nie ma

Reply:
A dla Springa i transakcji?

Reply:
RuntimeException powoduje wycofanie transakcji, a Exception nie